profiles/sap-hana-kvm-guest/tuned.conf

```diff
@@ -0,0 +1,18 @@
+#
+# tuned configuration
+#
+[main]
+summary=Optimize for running SAP HANA on KVM inside a virtual guest
+include=sap-hana
+
+[haltpoll]
```

Comment:
Maybe better to use the modules plugin, e.g.:
```
[modules]
cpuidle-haltpoll=+r force
```
The `+r` special flag tells TuneD to remove this module (if it is loaded) and reload it with the `force` parameter. In case the module is not loaded it's loaded with the `force` parameter.

And increase priority of the plugin instances which uses it, i.e. `[sysfs]` to be processed later:
```
[sysfs]
priority=10
....
```
Example e.g.:
https://github.com/redhat-performance/tuned/blob/master/profiles/balanced/tuned.conf